src/Symfony/Bundle/FrameworkBundle/Console/Descriptor/Descriptor.php

```diff
@@ -132,6 +135,11 @@ protected function write(string $content, bool $decorated = false)
      */
     abstract protected function describeContainerServices(ContainerBuilder $builder, array $options = []);
 
+    /**
+     * Describes container deprecations.
+     */
```

Comment:
Can be removed.

Reply:
I can remove it but all the other `describeContainerXXX` have a short description that mirror the method name.

```php
    /**
     * Describes a service definition.
     */
    abstract protected function describeContainerDefinition(Definition $definition, array $options = []);

    /**
     * Describes a service alias.
     */
    abstract protected function describeContainerAlias(Alias $alias, array $options = [], ContainerBuilder $builder = null);

    /**
     * Describes a container parameter.
     */
    abstract protected function describeContainerParameter($parameter, array $options = []);
```

@chalasr Should I remove all the redundant description or only the description for the new `describeContainerDeprecations()` method ?

Reply:
Only the new one please. The remaining should be removed on the lowest maintained branch where they exist.

Reply:
Ok done :)